src/main/java/agregator/io/LocalNews.java

```diff
@@ -15,15 +17,14 @@
 import agregator.structure.NewsItem;
 
 public class LocalNews {
-    public static List<NewsItem> parse(String path) throws ParserConfigurationException, SAXException, IOException {
+    public static List<NewsItem> parse(ServletContext context) throws ParserConfigurationException, SAXException, IOException {
```

Comment:
What happens  if I decide to use some config object or DB for storing locations?